spec/hydra/works/models/generic_work_spec.rb

```diff
@@ -13,15 +13,22 @@
 
   describe '#generic_works=' do
     it 'should aggregate generic_works' do
-      generic_work1.generic_works = [generic_work2, generic_work3]
-      expect(generic_work1.generic_works).to eq [generic_work2, generic_work3]
+      generic_work1.child_generic_works = [generic_work2, generic_work3]
+      expect(generic_work1.child_generic_works).to eq [generic_work2, generic_work3]
     end
   end
 
-  describe '#generic_files=' do
+  describe '#child_generic_files=' do
```

Comment:
I agree: it makes a lot of things simpler if you use GenericWork to model the hierarchy, and use GenericFile only as a leaf node to contain Files.  That would simplify the API a bit:

```
generic_work#child_generic_works
generic_work#parent_generic_works
generic_work#generic_files
generic_file#generic_works
```

But I'm pretty sure we included parent/child generic_file relationships because somebody wanted to use it.  So I'm reluctant to change that without broader input and consultation.  Should we send something to hydra-tech?  Maybe @mjgiarlo or @jpstroop remember if someone advocated for GenericFile hierarchy?